app/src/main/java/com/oracle/javafx/scenebuilder/app/SceneBuilderApp.java

```diff
@@ -496,8 +498,25 @@ private void createEmptyDocumentWindow() {
         newWindow.updateWithDefaultContent();
     }
 
+    /**
+     * By default all necessary actions to open a single file or a group of files take place in this method.
+     * If it is required to perform certain actions after successfully loading all files, please use {@code handleOpenFilesAction(List<String> files, Runnable onSuccess)} instead.
+     * 
+     * All error handling takes place here within, there is no way yet to access exceptional results and to work with them.
+     */
     @Override
     public void handleOpenFilesAction(List<String> files) {
+        handleOpenFilesAction(files, null);
```

Comment:
Please can we avoid passing `null` into methods (here and in the entire codebase, unless we are forced to by the API) and treat any method parameter as non-nullable by default. So here we could pass `() -> {}`.

Reply:
Agree, thats now implemented. Very good proposal, thank.